pyi.py

```diff
@@ -48,6 +42,19 @@ def unparse(node: ast.AST) -> str:
 __version__ = "22.8.0"
 
 LOG = logging.getLogger("flake8.pyi")
+FLAKE8_MAJOR_VERSION = flake8.__version_info__[0]
+
+if FLAKE8_MAJOR_VERSION < 5:
+    import warnings
+
+    warnings.warn(
+        (
+            "flake8-pyi will drop support for running with flake8 < 5.0.0 "
+            "in a future version. This will not happen until November 2022 "
+            "at the earliest."
+        ),
+        category=FutureWarning,
+    )
```

Comment:
I manually verified that this warning is emitted if the plugin is run in an environment with `flake8==4.x` installed. Other than that, the test suite still passes for me locally with `flake8==4.x` installed.